randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -1009,28 +1074,42 @@ Extra - room_id: [27]
   > Door to Entrance Lobby
       Any of the following:
           All of the following:
-              # Climb on bars
-              Charge Beam or Missiles ≥ 10
+              # Climb on bars: https://youtu.be/xEiqL9Vmaj4
               Any of the following:
+                  # Get past sidehoppers
+                  Charge Beam or Missiles ≥ 10
+                  Damage Boosts (Advanced) and Normal Damage ≥ 150
+              Any of the following:
+                  # Heat Reqs
                   Varia Suit
-                  Damage Runs (Intermediate) and Heat Damage ≥ 200
+                  Damage Runs (Intermediate) and Heat Damage ≥ 100
           All of the following:
-              # Shinespark
-              Speed Booster and Shinespark Tricks (Intermediate) and Disabled Door Lock Rando and Disabled Entrance Rando
+              # Shinespark Across https://youtu.be/QASI5KxDmj8
+              Speed Booster and Shinespark Tricks (Beginner) and Disabled Door Lock Rando and Disabled Entrance Rando
```

Comment:
this needs the blocks to be destroyed first. probably needs some kind of event since they never respawn.